### ISSUE TITLE
Is there currently a documented way to write cmdlets with boo?

### ISSUE THREAD

**Issue by zenware** [boo-lang/boo#162]
I'm interested in using PowerShell cmdlets in a work environment but I'm hoping for a more legible API and language than what's already provided by C#, VB, and the likes.

**Comment by masonwheeler** [comment]
I don't think we have any special functionality for that, but your stated goals sound like the sort of thing that Boo's metaprogramming functionality would help with.  Do you have any examples of what you're trying to do?
